misc/python-constraint/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "python-constraint" %}
```

Comment:
is this file autogenerated with something like `conda skeleton pypi {packagename}`? I actually never used it before, but if this is the result, then it's pretty good

Reply:
Yeah, but I had to edit a bit, and change from .gz to .bz2.